microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -444,9 +444,15 @@ def _install(self) -> bool:
                 dir_name=self._dpdk_repo_path_name,
             )
             if not self._dpdk_branch:
+
+                def filter_for_version_tags(tags: List[str]) -> List[str]:
+                    return [x for x in tags if x.startswith("v")]
```

Comment:
Can it just pass in a regexp?

Reply:
Sorry, I think the string of regexp is simpler. So the caller just need to pass in the string, not `re.compile(...`